### ISSUE TITLE
Do poprawy

### ISSUE THREAD

**Issue by kacgrzes** [mraq1234/submod4.4#1]
- [ ] czy komponent `Title` potrzebuje całej informacji na temat listy `todos` czy jedynie tego ile elementów znajduje się w tablicy (`todos.length`)?

**Comment by mraq1234** [comment]
Oczywiście, że nie potrzebuje sam na to wpadłem i w repozytorium do submodułu 4.5 jest już poprawione:
https://github.com/mraq1234/submod4.5
